openvino_wrapper_lib/src/inputs/standard_camera.cpp

```diff
@@ -18,6 +18,14 @@
  */
 #include "openvino_wrapper_lib/inputs/standard_camera.hpp"
 
+
+Input::StandardCamera::StandardCamera(const std::string & dev)
+{
+  if (!dev.empty() && isdigit(dev.back())) {
```

Comment:
as best, if dev.empty(), add an algro to calc the camera_id_?

Reply:
if dev.empty(),  the camera_id_ will calc in getCameraId() function, it will scan /dev/videoX to find a standart camera